src/games/behavmixed.imp

```diff
@@ -234,7 +187,7 @@ void MixedBehaviorProfile<T>::UndefinedToCentroid()
     GamePlayer player = efg->GetPlayer(pl);
     for (int iset = 1; iset <= player->NumInfosets(); iset++) {
       GameInfoset infoset = player->GetInfoset(iset);
-      if (GetInfosetProb(infoset) > (T) 0) {
+      if (GetInfosetProb(infoset) > T(0)) {
 	continue;
       }
       T total = (T) 0;
```

Comment:
It'll be useful to try to catch all the remaining C-style casts like this while we're at it.